### ISSUE TITLE
GradleのDEFAULT_JVM_OPTSを考える

### ISSUE THREAD

**Issue by disc99** [javamas/araignee#52]
## Context

Gradleで使用するJVMオプションをどうするか決めておく必要がある
## Problem, Idea

実行環境で必要な、JVMオプションが不足する
## Solution, next step

 GradleのDEFAULT_JVM_OPTSを決める

**Comment by dodoichi** [comment]
Gradleが実行されるVMに渡す引数という認識であってる？

目的は、OutOfMemoryにならないようにとか？

**Comment by disc99** [comment]
> 目的は、OutOfMemoryにならないようにとか？

そうですね、#51 のコメントにあったので、ひとまず作成しましたがデフォルトのままで使用しても恐らく問題になることは無いと思うので、特に用途が無ければCloseこのままします！

**Comment by dodoichi** [comment]
GCログを出しておくといいかもね。
